client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -386,6 +412,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 			onCheckout={ goToCheckout }
 			premiumBadge={ <PremiumBadge isPremiumThemeAvailable={ isPremiumThemeAvailable } /> }
 			heading={ heading }
+			blankCanvas={ blankCanvasContent }
```

Comment:
We could avoid renaming variables when passing them for more clarity, use `blankCanvasContent={ blankCanvasContent }` or `blankCanvasCta={ blankCanvasCta }`.